packages/discord.js/src/structures/StageInstance.js

```diff
@@ -83,6 +93,15 @@ class StageInstance extends Base {
     return this.client.guilds.resolve(this.guildId);
   }
 
+  /**
+   * The associated guild scheduled event of this stage instance
+   * @type {?GuildScheduledEvent}
+   * @readonly
+   */
+  get scheduledEvent() {
```

Comment:
I think you should either call this guildScheduledEvent or call the other prop scheduledEventId so their names match